stix2/test/test_custom.py

```diff
@@ -128,6 +128,40 @@ def test_custom_observable_object():
         NewObservable(property1='something', property2=4)
 
 
+def test_custom_no_properties_raises_exception():
+    with pytest.raises(ValueError):
```

Comment:
Will all five of these custom objects create errors? If so, there should be five separate tests. If not, the ones that don't should be outside the `pytest.raises` block (and also likely in their own tests anyway)

Reply:
Yep. I removed some of them in a later commit since they were covering the same area. I divided them into individual test cases and made sure the code coverage was good for each of the cases.